server/jetstream_cluster_4_test.go

```diff
@@ -1349,6 +1350,7 @@ func TestJetStreamClusterStreamOrphanMsgsAndReplicasDrifting(t *testing.T) {
 			ldmRestart:     true,
 			rolloutRestart: false,
 			restarts:       1,
+			checkHealthz:   true,
```

Comment:
Getting similar results with `checkHealthz` enabled here but we can give it a try to add since using healthz would be closer to what a usual rollout would look like.